python/repositories.bzl

```diff
@@ -144,6 +144,12 @@ filegroup(
         allow_empty = True,
         exclude = [
             "**/* *", # Bazel does not support spaces in file names.
+            # static libraries
+            "lib/*.a",
```

Comment:
```suggestion
```

Reply:
The glob below captures this one.

Reply:
Thanks for the suggestion. I created #771 for this change.